packages/slave/slave.js

```diff
@@ -692,6 +694,8 @@ class Instance extends libLink.Link {
 	}
 
 	async masterConnectionEventEventHandler(message) {
+		this.notifyStatus(this._status);
+
```

Comment:
Why are you doing this? It'll send updates when the connection is down as well as unnecessary updates when the connection is dropped and resumed.